tsdb/chunkenc/histogram.go

```diff
@@ -45,6 +45,28 @@ func NewHistogramChunk() *HistogramChunk {
 	return &HistogramChunk{b: bstream{stream: b, count: 0}}
 }
 
+func NewPrepopulatedHistogramChunk(b []byte) (*HistogramChunk, error) {
```

Comment:
Could you add a doc comment what this is doing?

Reply:
Yep, will do!